optimism/src/lookups.rs

```diff
@@ -71,7 +72,54 @@ pub(crate) trait FixedLookupTables<F> {
     fn table_byte() -> LookupTable<F>;
 }
 
-impl<F: Field> FixedLookupTables<F> for LookupTable<F> {
+impl<F: Field> FixedLookupTables<F, LookupTableIDs> for LookupTable<F> {
+    fn in_table(id: LookupTableIDs, value: Vec<F>) -> Option<usize> {
```

Comment:
I would rename it into `is_in_table`, and add a small line of documentation.

Reply:
Oh sure, forgot about that!